src/main/webapp/scripts/session-script.js

```diff
@@ -31,6 +51,51 @@ function addOnClickListenerToElements() {
   });
 }
 
+/**
+ * function buildAttendeeDiv() adds the div element containing
+ * all the elements representing an attendee to the session info
+ * attendees div.
+ * @param {string} nameOfAttendee name of attendee to build
+ * @param {string} controller name of the controller of the session
+ */
+function buildAttendeeDiv(nameOfAttendee, controller) {
+  const /** HTMLElement */ sessionInfoAttendeesDiv =
+      document.getElementById('session-info-attendees');
+  const /** HTMLDivElement */ attendeeDiv = document.createElement('div');
+  attendeeDiv.className = 'attendee-div'
+  const /** HTMLSpanElement */ controllerToggle = 
+      document.createElement('span');
+  controllerToggle.className = 'controller-toggle';
+  controllerToggle.addEventListener('click', event => {
+    changeControllerTo(event, controller);
+  }, /**AddEventListenerOptions=*/false);
+  const /** HTMLHeadingElement */ attendeeName =
+      document.createElement('h3');
+  attendeeName.innerHTML = nameOfAttendee;
+  attendeeName.className = 'attendee-name'
+  attendeeName.id = nameOfAttendee;
+  attendeeDiv.appendChild(controllerToggle);
+  attendeeDiv.appendChild(attendeeName);
+  sessionInfoAttendeesDiv.appendChild(attendeeDiv);
+}
+
+/**
+ * If the current controller of the session clicks on the controller 
+ * toggle, their controller status is revoked and the server is updated
+ * with information on the new controller.
+ * @param {MouseEvent} event the event that captures what was clicked on
+ * @param {string} controller name of the controller of the session
+ */
+function changeControllerTo(event, controller) {
+  if (urlParameters.get('name') === controller) {
```

Comment:
could you make a global, constant javascript object that represents the expected url paramters in the page?

const URL_PARAM_KEY = {
SCREEN_NAME : 'name',
SESSION_ID : 'session_id'
};

So now we have:
urlParamters.get(URL_PARAM_KEY.SCREEN_NAME)